### DISCUSSION TITLE
help with ArduinoVNC

### DISCUSSION THREAD

**Discussion by Hooba97** [moononournation/Arduino_GFX#370] [Q&A]
Hello I am trying to connect my 7" screen (ZX7D00CE01S) to my Raspberry Pi, I followed the instructables step by step [https://www.instructables.com/Arduino-Thin-Client/](url)
i got it to connect to the server but the screen isn't displaying the remote 
its only displaying 

"Connect VNC
192.168.6.22:5900"

the serial monitor returns this on repeat   : 

!connected
01:05:06.880 -> [rfb_connect_to_server] Connected.
01:05:06.880 -> [_rfb_negotiate_protocol] Server protocol: RFB 003.008
01:05:06.880 -> [_rfb_negotiate_protocol] used protocol: RFB 003.008
01:05:06.880 -> 
01:05:06.880 -> Authentication OK
01:05:06.880 -> [VNC-SERVER] VNC Server config - Name: iehab's X desktop (raspberrypi:1)
01:05:06.927 ->  - width:800      height:480
01:05:06.927 ->  - bpp:32        depth:24       bigendian:0     truecolor:1
01:05:06.927 ->  - redmax:255     greenmax:255    bluemax:255
01:05:06.927 ->  - redshift:16   greenshift:8  blueshift:0
01:05:06.927 -> [VNC-CLIENT] Supported Encodings:
01:05:06.927 ->  - ZRLE
01:05:06.927 ->  - Hextile
01:05:06.927 ->  - RRE
01:05:06.927 ->  - CoRRE
01:05:06.927 ->  - Raw
01:05:06.927 -> [VNC-CLIENT] Supported Special Encodings:
01:05:06.927 ->  - SetDesktopSize
01:05:06.927 ->  - CursorPos
01:05:06.927 ->  - ContinuousUpdates
01:05:06.927 ->  - compresslevel: 4
01:05:06.927 -> [VNC-CLIENT] Client pixel format:
01:05:06.927 ->  - width:800      height:480
01:05:06.927 ->  - bpp:16        depth:16        bigEndian:0    trueColor:1
01:05:06.927 ->  - red-max:31    green-max:63    blue-max:31
01:05:06.974 ->  - red-shift:11  green-shift:5  blue-shift:0
01:05:06.974 -> vnc_connect Done.

please help

**Comment by moononournation** [comment]
may be try align the server parameters as client

**Reply by Hooba97** [reply]
I got it to work using a different vnc server 
tigervnc works so far happy to keep you updated if you want
now i need to figure out touch controls

**Comment by Hooba97** [comment]
can you help me with TAMCTec GT911 touch library , the commands are different

**Reply by moononournation** [reply]
TouchLib also support GT911

**Reply by Hooba97** [reply]
it doesn't work for some reason, even the example file in TouchLib
after staying up all night i did it and everything is working

**Reply by Thomas-Busch-Waterloo** [reply]
How did you get it working?
For me even in the TouchCalibration, it works for the first touch...takes a few tries then it stops responding...and sometimes just reboots a few times.
